### ISSUE TITLE
About different image sizes in Table 2

### ISSUE THREAD

**Issue by zmhhmz** [YouHuang67/InterFormer#1]
In Table 2 of the paper, you adopt different image sizes for different methods (with 512 for InterFormer). Would it be unfair for the performance (NoC) comparison?

**Comment by YouHuang67** [comment]
Indeed, the different image sizes being discussed are the default settings
for various methods, and changing these settings might negatively affect
performance. So, it's important to use the specific settings for each
method. Again, the resize strategy is not always better when larger,
especially as the resize process may cause object deformation. An intuitive
example is resizing the ground truth mask to twice its size (using the
nearest strategy) and then resizing it back to its original size, which may
result in shape changes. Therefore, it is necessary to use the default
settings.

MinghaoZhou ***@***.***> 于2023年4月17日周一 19:20写道：
